src/components/common/datePicker/DateRangePicker.tsx

```diff
@@ -160,10 +163,13 @@ export const DatePickerButton = memo(
           <CalendarIcon className="h-12 w-12" />
         </span>
 
-        <span className="justify-self-end text-14 text-white/80">
+        <span
+          className="justify-self-end text-14 text-white/80"
+          data-testid="simulation-dates"
+        >
           {hasDates ? (
             <>
-              {startDate} – {endDate}
+              {startDate} - {endDate}
```

Comment:
I think this was the expected character

Reply:
It's an odd character to use, that's why I removed it. Re-added now and to the testcases as well